### PR TITLE
Ran the upgrade version script for you

### PR DESCRIPTION
**Pull request by VoR0220**
I added a couple lines because I saw it wasn't upgrading the version in certain python scripts that might now have the version needed. When I ran it it seemed to update the docs just fine? Adding them in to the userGuides.

### CODE REVIEW

File: test/util/TestCommonFoundry.sol [hunk 1]
```diff
@@ -735,7 +735,7 @@ abstract contract TestCommonFoundry is TestCommon, EndWithStopPrank, EnabledActi
         erc721.connectHandlerToToken(address(handler));
         /// register the token
         applicationAppManager.registerToken(name, address(erc721));
-        HandlerVersionFacet(address(handler)).updateVersion("2.1.0");
+        HandlerVersionFacet(address(handler)).updateVersion("2.2.0");
```
Comment (ShaneDuncan602) [inline test/util/TestCommonFoundry.sol:738]: I don't see where this was added to the versioning script. Was this already being changed and just got missed?

Comment (TJ-Everett) [reply test/util/TestCommonFoundry.sol:738]: +1 to this Question.

**Review by TJ-Everett — COMMENTED** [review]
Just a comment about if an updated file was missed or needs to be added to the script. 
Also just a psa that we want to ensure that our commit messages are applicable to the changes we make within the scope of the ticket. That way we can reference back to these and know what was changed within the commit.

File: test/util/TestCommonFoundry.sol (hunk 1 above)
Comment (VoR0220) [reply test/util/TestCommonFoundry.sol:738]: No I just ran the script and this was the output? I'm confused myself. The script works on my local.

Comment (ShaneDuncan602) [reply test/util/TestCommonFoundry.sol:738]: Hmm, might have been a merge issue that was never detected.

**Review by ShaneDuncan602 — APPROVED** [review]
Good job!

**Review by mpetersoCode55 — APPROVED** [review]

**Pull request merged**